samples/unpeelingOnion/src/onion/ecommerce/application/shoppingCarts/commands/shoppingCarts/openShoppingCart.ts

```diff
@@ -0,0 +1,10 @@
+import { Command } from '#core/commands';
+
+export class OpenShoppingCart extends Command {
```

Comment:
In TS interface is as well treated as data class.

Reply:
Yup, that’s the intention of this sample. Classes are used as 1:1 ports of C#/Java habits. In the unpeeled version, I got rid of them, showing that there’s another way in TS 🙂